Whisper.net/WhisperProcessorEvents.cs

```diff
@@ -15,6 +15,35 @@ public class EncoderBeginData
 
 }
 
+/// <summary>
+/// Represents data about a recognized token.
+/// </summary>
+/// <param name="id"></param>
+/// <param name="Tid"></param>
+/// <param name="P"></param>
+/// <param name="Plog"></param>
+/// <param name="Pt"></param>
+/// <param name="PtSum"></param>
+/// <param name="T0"></param>
+/// <param name="T1"></param>
+/// <param name="T_Dtw"></param>
+/// <param name="Vlen"></param>
+/// <param name="text"></param>
+public class WhisperToken
+{
+    public int Id;
+    public int Tid;
+    public float P;
+    public float Plog;
+    public float Pt;
+    public float PtSum;
+    public long T0;
```

Comment:
As these are the managed properties that our users will be able to read directly, can we make them a little friendlier to dotnet world?

e.g. `long T0` =>`TimeSpan StartTime` so that they don't need to know too much of whisper internals to read this properties (e.g. what T0 represents).